shared/src/business/useCases/getCaseDeadlinesInteractor.test.ts

```diff
@@ -170,108 +172,4 @@ describe('getCaseDeadlinesInteractor', () => {
       startDate: START_DATE,
     });
   });
-
-  it('logs an error for any case that fails validation and includes all cases that pass validation', async () => {
```

Comment:
We are returning a subset of data, not a case. Validating it the same way felt incorrect.